### PR TITLE
Feature/add ideas functionality m

### PR DESCRIPTION
**Pull request by minsungpark89**
What (if any) features are you implementing?
- Added functionality to disable and enable the saveButton button.  This allows a user to input values in the title and body to create a new card.
What (if anything) did you refactor?
- refactored by adding one CSS rule and a HTML attribute.  WIP still
Were there any issues that arose?
- no this is a working branch.  
Is there anything that you need from your teammate?
- renaming variables maybe.
Any other comments, questions, or concerns?
Good job team!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -102,7 +102,7 @@ <h2 class="idea-title">Idea title</h2>
       </article>
     </section>
   </main>
-    <script src="./src/idea.js"></script>
-    <script src="./src/main.js"></script>
+  <script src="./Idea.js"></script>
```
Comment (JRicardoT) [inline index.html:105]: Good catch. That source would've gave us problems.

File: main.js [hunk 2]
```diff
@@ -1 +1,65 @@
...
+      </article>`
+  }
+};
+
+function clearInput() {
+  titleInput.value = '';
+  bodyInput.value = '';
+};
+
+function checkInputFields() {
```
Comment (JRicardoT) [inline main.js:49]: Maybe we could change the name ??

File: styles.css [hunk 3]
```diff
@@ -99,7 +98,12 @@ label {
   height: 2.5em;
   margin-top: 1.5em;
   border: none;
+  cursor: pointer;
 }
+/*
```
Comment (JRicardoT) [inline styles.css:103]: Have to delete this comment.

**Review by JRicardoT — APPROVED** [review]
Aside from have to rename some function this looks good. Ready to merge.

File: index.html (hunk 1 above)
Comment (tanya2108) [inline index.html:105]: Great work, finding this error.

File: main.js [hunk 4]
```diff
@@ -1 +1,65 @@
...
+  if (titleInput.value && bodyInput.value) {
```
Comment (tanya2108) [inline main.js:50]: Really like this simplified from the ".value ===  "

**Review by tanya2108 — COMMENTED** [review]
Great job! Ready to merge.

**Pull request merged**